superset/assets/src/utils/DebouncedMessageQueue.js

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
I extract the message queue logic and create a class called `DebouncedMessageQueue`.

Reply:
🥇